src/components/static-map.js

```diff
@@ -141,6 +144,9 @@ export default class StaticMap extends PureComponent {
       canvas.style.outline = 'none';
     }
 
+    // Attach optional onLoad function
+    map.once('load', () => this.props.onLoad());
```

Comment:
I was confused by the previous code wrapping onLoad too. From the previous implementation and unit test, not exposing the map object was desirable. Maybe this was a work around for a previous version?

It feels cleaner to do `map.once('load', this.props.onLoad);`. I was trying to retain original behavior with the wrapper. @Pessimistress @abmai ?